### PR TITLE
Adds a suicide for the replica katana

### PR DESCRIPTION
**Pull request by ghost**
I like finding all the funny suicide acts, but there just isn't that many in the game right now! That's about to change!

### CODE REVIEW

**Comment by Armhulen** [comment]
what the fuck

**Comment by Armhulen** [comment]
oh whoops

**Comment by Cyberboss** [comment]
The PR is fine, just fix your master

File: code/game/objects/items/toys.dm [hunk 1]
```diff
@@ -347,6 +347,9 @@
 	attack_verb = list("attacked", "slashed", "stabbed", "sliced")
 	hitsound = 'sound/weapons/bladeslice.ogg'
 
+/obj/item/toy/katana/suicide_act(mob/user)
```
Comment (Cyberboss) [inline code/game/objects/items/toys.dm]: mob/living

File: code/game/objects/items/toys.dm [hunk 2]
```diff
@@ -347,6 +347,9 @@
...
+	user.visible_message("<span class='suicide'>[user] slices the air around [user.p_them()] furiously with [src]! It looks like [user.p_theyre()] trying to unleash their final move!</span>")
+	return(BRUTELOSS, FIRELOSS)
```
Comment (Cyberboss) [inline code/game/objects/items/toys.dm]: return BRUTELOSS | FIRELOSS

**Review by Cyberboss — CHANGES_REQUESTED** [review]

**Comment by Armhulen** [comment]
i already moved this to another one so can we do that oh good

**Pull request closed without merging**